pkg/server/endpoints/config.go

```diff
@@ -3,17 +3,26 @@ package endpoints
 import (
 	"net"
 	"net/url"
-	"sync"
 
-	observer "github.com/imkira/go-observer"
 	"github.com/sirupsen/logrus"
 	"github.com/spiffe/spire/pkg/common/telemetry"
 	"github.com/spiffe/spire/pkg/server/ca"
 	"github.com/spiffe/spire/pkg/server/catalog"
+	"github.com/spiffe/spire/pkg/server/svid"
 
 	"google.golang.org/grpc"
 )
 
+type SVIDRotator interface {
```

Comment:
Would this be better defined in the `svid` package? `svid.Consumer`? `svid.Observer`?

Reply:
sure, i like that.